include/bluetooth/services/cts_client.h

```diff
@@ -66,22 +82,41 @@ typedef void (*bt_cts_read_cb)(struct bt_cts_client *cts_c,
 /**@brief Value notification callback.
  *
  * This function is called every time the server sends a notification
- * for a changed value.
+ * for a changed value of the Current Time.
  *
  * @param cts_c         Current Time Service client structure.
  * @param current_time  The notified current time value.
  */
 typedef void (*bt_cts_notify_cb)(struct bt_cts_client *cts_c,
 				 struct bt_cts_current_time *current_time);
 
-/**@brief Current Time Service client structure. This structure contains status information for the client. */
+/**@brief Read complete callback.
+ *
+ * This function is called when the Local Time read operation finishes.
+ *
+ * @param cts_c         Local Time Service client structure.
+ * @param local_time	The local time value that was read.
+ * @param err           Error code.
+ */
+typedef void (*bt_lts_read_cb)(struct bt_cts_client *cts_c,
+			       struct bt_cts_local_time *local_time,
```

Comment:
Can it be const ?

Reply:
Followed the convention for:
typedef void (*bt_cts_notify_cb)(struct bt_cts_client *cts_c, struct bt_cts_current_time *current_time);